resolwe_bio/tests/processes/test_support_processors.py

```diff
@@ -548,3 +548,23 @@ def test_umi_tools_dedup(self):
 
         dedup_paired = self.run_process('umi-tools-dedup', {'alignment': bam_paired.id})
         self.assertFile(dedup_paired, 'stats', './corall/output/dedup_paired_stats.txt')
+
+    @tag_process('seqtk-rev-complement-single', 'seqtk-rev-complement-paired')
+    def test_reverse_complement(self):
+        with self.preparation_stage():
+            single_reads = self.prepare_reads(['hs_slamseq_R1.fastq.gz'])
+
+            paired_reads = self.prepare_paired_reads(['hs_slamseq_R1.fastq.gz'], ['hs_slamseq_R2.fastq.gz'])
+
+            revcomp_single = self.run_process('seqtk-rev-complement-single', {'reads': single_reads.id})
```

Comment:
This should be one indentation level less than it is. The `preparation_stage` context manager only prepares input data objects, but the process that is tested should be put outside of it (and also all assert statements that follow it).